docs/maintainers/concept-exercise-style-guide.md

```diff
@@ -0,0 +1,56 @@
+# Formatting and Style Guide for Concept Exercise Files
+
+This document records the current styleguide applicable to Concept exercise Markdown files, which refer to the files contained in `.docs` and `.meta`. Please follow them when implementing a Concept Exercise; maintainers should also reference the Style Guide when reviewing pull requests. When the relevant language also includes a style guide, those preferences will take precedence over the ones here.
+
+## Headers
+
+ - Use level 2 headers, e.g. `##`
+ - Do not use top-level headers, e.g. `#`
+ - Do not use leading whitespace in the file.
+
+
+## Empty Documents
+
+If a document is empty, i.e. there is no `after.md` content, then do not check one in.
+
+
+## Links
+
+Exercism prefers "indirect reference" links. They are defined at the bottom of the Markdown file and mapped to a reference slug, and referenced by that slug in the text.
+
+This makes maintenance easier, since the link must only be updated once.
+
+Example:
+
+```
+I have a paragraph of text that links to the same page twice. 
+
+The [first link][indirect-reference] in one sentence. 
+
+Then, another sentence with the [link repeated][indirect-reference].
+
+[indirect-reference]: https://example.com/link-to-page
+```
+
+Links should be inline where possible, i.e. as `<a href="">text</a>` tags instead of bare URLs. This is accomplished using the Markdown bracketed notation, as exemplified above.
+
+
+## Code
+
+Where possible and relevant, format code first as if it was being typed into a terminal, or if not applicable to your language, as if in a code file itself. Please reference the docs for your language, as they may have more information.
```

Comment:
I think this would specifically be for examples of code that **we** will run. ie if we show an example of a function definition, I wouldn't want this, but if we were giving an example of the input/output of tests, then I would agree with this.

Reply:
Agreed. So there are two cases for examples:

1. Code samples to illustrate how something works in a language (usually defined in the `introduction.md` file). These don't need any terminal prompt and such, as that will probably only distract from the actual code.
1. Code samples to show what a piece of code (usually a function/method) returns (usually defined in the `instructions.md` file). These examples would indeed benefit from the terminal prompt bit.